CONTRIBUTING.md

```diff
@@ -111,13 +120,12 @@ If an issue is a `bug`, and it doesn't have a clear reproduction that you have p
 
 - `bug`s should be closed when the issue is fixed and merged to `master`
```

Comment:
Although maybe it's easier for us to ignore `fixed` issues, and we can use them to construct a meaningful changelog. I change my vote ;)